version1/img/archive-model-four.graphml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:java="http://www.yworks.com/xml/yfiles-common/1.0/java" xmlns:sys="http://www.yworks.com/xml/yfiles-common/markup/primitives/2.0" xmlns:x="http://www.yworks.com/xml/yfiles-common/markup/2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:y="http://www.yworks.com/xml/graphml" xmlns:yed="http://www.yworks.com/xml/yed/3" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://www.yworks.com/xml/schema/graphml/1.1/ygraphml.xsd">
-  <!--Created by yEd 3.17.2-->
+  <!--Created by yEd 3.19-->
```

Comment:
The changes in the model diagrams are misleading.

You have linked `Dokumentversjon` to the box that contains both `EksternId` and `Noekkelord`. Furthermore, it implies that there may be other such entities. Since `Dokumentversjon` cannot be linked to `Noekkelord` this information is misleading.

I suggest reverting these changes and adding an additional box containing `EksternId` to which `Dokumentversjon` can be linked.

Reply:
Addressed in 585239d